### PR TITLE
Renaming ahoy.

### PR DESCRIPTION
**Pull request by Shadeykins**
## About The Pull Request

Same as #568, but without accidentally nuking everything.

## Changelog
:cl:
add: Horizons/RRT/Head Office/ICC
del: CC/NT/ERT
/:cl:

### CODE REVIEW

File: code/modules/projectiles/pins.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/item/firing_pin
 	name = "electronic firing pin"
-	desc = "A small authentication device, to be inserted into a firearm receiver to allow operation. NT safety regulations require all new designs to incorporate one."
+	desc = "A small authentication device, to be inserted into a firearm receiver to allow operation. safety regulations require all new designs to incorporate one."
```
Comment (AffectedArc07) [inline code/modules/projectiles/pins.dm]: Capital `S` please

File: code/modules/ruins/spaceruin_code/oldstation.dm [hunk 2]
```diff
@@ -25,7 +25,7 @@
 	name = "K14 Energy Gun Report"
 	info = "<b>*K14-Multiphase Energy Gun*</b><br><br>The K14 Prototype Energy Gun is the first Energy Rifle that has been successfully been able to not only hold a larger ammo charge \
 	than other gun models, but is capable of swapping between different energy projectile types on command with no incidents.<br><br>The weapon still suffers several drawbacks, its alternative, \
-	non laser fire mode, can only fire one round before exhausting the energy cell, the weapon also remains prohibitively expensive, nonetheless NT Market Research fully believe this weapon \
+	non laser fire mode, can only fire one round before exhausting the energy cell, the weapon also remains prohibitively expensive, nonetheless Market Research fully believe this weapon \
```
Comment (AffectedArc07) [inline code/modules/ruins/spaceruin_code/oldstation.dm]: Does `Market Research` need to be capitalised here

File: code/modules/ruins/spaceruin_code/oldstation.dm [hunk 3]
```diff
@@ -45,7 +45,7 @@
 	name = "Crew Reawakening Report"
 	info = "Artificial Program's report to surviving crewmembers.<br><br>Crew were placed into cryostasis on March 10th, 2445.<br><br>Crew were awoken from cryostasis around June, 2557.<br><br> \
 	<b>SIGNIFICANT EVENTS OF NOTE</b><br>1: The primary radiation detectors were taken offline after 112 years due to power failure, secondary radiation detectors showed no residual \
-	radiation on station. Deduction, primarily detector was malfunctioning and was producing a radiation signal when there was none.<br><br>2: A data burst from a nearby Nanotrasen Space \
+	radiation on station. Deduction, primarily detector was malfunctioning and was producing a radiation signal when there was none.<br><br>2: A data burst from a nearby Space \
```
Comment (AffectedArc07) [inline code/modules/ruins/spaceruin_code/oldstation.dm]: Does `Space Station` need to be capitalised here

File: modular_citadel/code/modules/client/loadout/__donator.dm [hunk 4]
```diff
@@ -430,7 +430,7 @@ datum/gear/darksabresheath
 	ckeywhitelist = list("maxlynchy")
 
 /datum/gear/csheet
-	name = "NT Bedsheet"
+	name = "Corporate Bedsheet"
```
Comment (AffectedArc07) [inline modular_citadel/code/modules/client/loadout/__donator.dm]: Make sure to update the icon for this

File: code/game/objects/effects/contraband.dm [hunk 5]
```diff
@@ -586,7 +586,7 @@
 
 /obj/structure/sign/poster/official/no_erp
 	name = "No ERP"
-	desc = "This poster reminds the crew that Eroticism, Rape and Pornography are banned on Nanotrasen stations."
+	desc = "This poster reminds the crew that English Breakfast Tea, Richard Simmons, and Parmesan are banned on Horizons stations."
```
[the remark this answers is not in the document]
Comment (Shadeykins) [reply code/game/objects/effects/contraband.dm]: You just want to preserve your Englishness.

File: code/modules/admin/secrets.dm [hunk 6]
```diff
@@ -229,10 +229,10 @@
 		if("moveferry")
 			if(!check_rights(R_ADMIN))
 				return
-			SSblackbox.record_feedback("nested tally", "admin_secrets_fun_used", 1, list("Send CentCom Ferry"))
+			SSblackbox.record_feedback("nested tally", "admin_secrets_fun_used", 1, list("Send Head Office Ferry"))
```
[the remark this answers is not in the document]
Comment (Shadeykins) [reply code/modules/admin/secrets.dm]: Typically fucking with ferry names in variables has never been an issue in my experience, but fixed.

File: code/modules/antagonists/traitor/IAA/internal_affairs.dm [hunk 7]
```diff
@@ -240,14 +240,14 @@
 	to_chat(owner.current, "<span class='userdanger'>You are the [special_role].</span>")
 	if(syndicate)
 		to_chat(owner.current, "<span class='userdanger'>Your target has been framed for [crime], and you have been tasked with eliminating them to prevent them defending themselves in court.</span>")
-		to_chat(owner.current, "<B><font size=5 color=red>Any damage you cause will be a further embarrassment to Nanotrasen, so you have no limits on collateral damage.</font></B>")
+		to_chat(owner.current, "<B><font size=5 color=red>Any damage you cause will be a further embarrassment to Horizons, so you have no limits on collateral damage.</font></B>")
```
[the remark this answers is not in the document]
Comment (Shadeykins) [reply code/modules/antagonists/traitor/IAA/internal_affairs.dm]: This is for the weird evil IAA agent events that encourages shitler IAAs to do the big bads.

File: code/modules/cargo/bounties/security.dm [hunk 8]
```diff
@@ -27,28 +27,28 @@
 
 /datum/bounty/item/security/sechuds
 	name = "Sec HUDs"
-	description = "Nanotrasen military academy has started to train officers how to use Sec HUDs to the fullest affect. Please send spare Sec HUDs so we can teach the men."
+	description = "The ICC military academy has started to train officers how to use Sec HUDs to the fullest affect. Please send spare Sec HUDs so we can teach the men."
```
[the remark this answers is not in the document]
Comment (Shadeykins) [reply code/modules/cargo/bounties/security.dm]: Fixed.

File: code/game/objects/structures/bedsheet_bin.dm [hunk 9]
```diff
@@ -214,8 +214,8 @@ LINEN BINS
 	dream_messages = list("a book", "an explosion", "lightning", "a staff", "a skeleton", "a robe", "magic")
 
 /obj/item/bedsheet/nanotrasen
-	name = "nanotrasen bedsheet"
-	desc = "It has the Nanotrasen logo on it and has an aura of duty."
+	name = "horizons bedsheet"
```
Comment (AffectedArc07) [inline code/game/objects/structures/bedsheet_bin.dm]: Icon

File: _maps/RandomRuins/SpaceRuins/listeningstation.dmm [hunk 10]
```diff
@@ -1036,7 +1036,7 @@
 /obj/effect/mob_spawn/human/lavaland_syndicate/comms/space{
 	assignedrole = "Space Syndicate";
 	dir = 8;
-	flavour_text = "<span class='big bold'>You are a syndicate agent,</span><b> assigned to a small listening post station situated near your hated enemy's top secret research facility: Space Station 13. <b>Monitor enemy activity as best you can, and try to keep a low profile. <font size=6>DON'T</font> abandon the base without good cause.</b> Use the communication equipment to provide support to any field agents, and sow disinformation to throw Nanotrasen off your trail. Do not let the base fall into enemy hands!</b>"
+	flavour_text = "<span class='big bold'>You are a syndicate agent,</span><b> assigned to a small listening post station situated near your an enemy research facility. <b>Monitor enemy activity as best you can, and try to keep a low profile. <font size=6>DON'T</font> abandon the base without good cause.</b> Use the communication equipment to provide support to any field agents, and sow disinformation to throw Horizons employees off your trail. Do not let the base fall into enemy hands!</b>"
```
Comment (Arturlang) [inline _maps/RandomRuins/SpaceRuins/listeningstation.dmm]: Should probably be near an enemy reseearch outpost, removing the your

File: code/__DEFINES/cargo.dm [hunk 11]
```diff
@@ -17,12 +17,12 @@
 #define POD_DESC 3
 
 #define POD_STYLES list(\
-    list("supplypod", "supply pod", "A Nanotrasen supply drop pod."),\
-    list("bluespacepod", "bluespace supply pod" , "A Nanotrasen Bluespace supply pod. Teleports back to CentCom after delivery."),\
-    list("centcompod", "\improper Centcom supply pod", "A Nanotrasen supply pod, this one has been marked with Central Command's designations. Teleports back to Centcom after delivery."),\
+    list("supplypod", "supply pod", "A supply drop pod."),\
+    list("bluespacepod", "bluespace supply pod" , "A bluespace supply pod. Teleports back to Head Office after delivery."),\
```
Comment (Arturlang) [inline code/__DEFINES/cargo.dm]: Theres a missing the. Teleports to the head office

File: code/controllers/subsystem/shuttle.dm [hunk 12]
```diff
@@ -205,7 +205,7 @@ SUBSYSTEM_DEF(shuttle)
 			to_chat(user, "The emergency shuttle is moving away to a safe distance.")
 			return
 		if(SHUTTLE_STRANDED)
-			to_chat(user, "The emergency shuttle has been disabled by CentCom.")
+			to_chat(user, "The emergency shuttle has been disabled by Head Office.")
```
Comment (Arturlang) [inline code/controllers/subsystem/shuttle.dm]: missing the here as well, though probably could work without

File: code/datums/shuttles.dm [hunk 13]
```diff
@@ -271,7 +271,7 @@
 /datum/map_template/shuttle/emergency/cramped
 	suffix = "cramped"
 	name = "Secure Transport Vessel 5 (STV5)"
-	description = "Well, looks like CentCom only had this ship in the area, they probably weren't expecting you to need evac for a while. \
+	description = "Well, looks like Head Office only had this ship in the area, they probably weren't expecting you to need evac for a while. \
 	Probably best if you don't rifle around in whatever equipment they were transporting. I hope you're friendly with your coworkers, because there is very little space in this thing.\n\
```
Comment (Arturlang) [inline code/datums/shuttles.dm]: missing a the here

File: code/game/gamemodes/revolution/revolution.dm [hunk 14]
```diff
@@ -195,7 +195,7 @@
 /datum/game_mode/revolution/generate_report()
 	return "Employee unrest has spiked in recent weeks, with several attempted mutinies on heads of staff. Some crew have been observed using flashbulb devices to blind their colleagues, \
 		who then follow their orders without question and work towards dethroning departmental leaders. Watch for behavior such as this with caution. If the crew attempts a mutiny, you and \
-		your heads of staff are fully authorized to execute them using lethal weaponry - they will be later cloned and interrogated at Central Command."
+		your heads of staff are fully authorized to execute them using lethal weaponry - they will be later cloned and interrogated at Head Office."
```
Comment (Arturlang) [inline code/game/gamemodes/revolution/revolution.dm]: missing the here

File: code/game/machinery/dance_machine.dm [hunk 15]
```diff
@@ -63,7 +63,7 @@
 		user.playsound_local(src,'sound/misc/compiler-failure.ogg', 25, 1)
 		return
 	if(!SSjukeboxes.songs.len)
-		to_chat(user,"<span class='warning'>Error: No music tracks have been authorized for your station. Petition Central Command to resolve this issue.</span>")
+		to_chat(user,"<span class='warning'>Error: No music tracks have been authorized for your station. Petition Head Office to resolve this issue.</span>")
```
Comment (Arturlang) [inline code/game/machinery/dance_machine.dm]: Missing the here

**Comment by Arturlang** [comment]
Checked the whole thing, relaly only one proper mistake and a lot of missing thes in front of head office, unsure if they should be there or not.

File: _maps/RandomRuins/SpaceRuins/listeningstation.dmm (hunk 10 above)
Comment (Shadeykins) [reply _maps/RandomRuins/SpaceRuins/listeningstation.dmm]: Fixed.

**Comment by Shadeykins** [comment]
Deconflicted. Hopefully nothing else explodes.

File: code/modules/projectiles/pins.dm (hunk 1 above)
Comment (AffectedArc07) [reply code/modules/projectiles/pins.dm]: This `S` on ` operation. safety ` is still lowercase.

**Pull request merged**